lib/icons/tag.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
Is this from the new Icon pack? Can this be added as a separate PR?

Reply:
Which new Icon pack? It's the tag icon shown next to a tag name (differentiates tag suggestions from search suggestions). Tyler shared the SVG with me in Slack.